src/components/Card-Chef_Dillon.js

```diff
@@ -0,0 +1,84 @@
+class Card {
+  constructor(
+    data,
+    cardSelector,
+    handleImageClick,
+    handleDeleteClick,
+    handleLikeClick
+  ) {
+    this._name = data.name;
+    this._link = data.link;
+    this._id = data._id;
+    this._isLiked = data.isLiked;
+    this._handleImageClick = handleImageClick;
+    this._handleDeleteClick = handleDeleteClick;
+    this._handleLikeClick = handleLikeClick;
+    this._cardSelector = cardSelector;
+  }
+
+  _getTemplate() {
+    const cardElement = document
+      .querySelector(this._cardSelector)
+      .content.firstElementChild.cloneNode(true);
+
+    return cardElement;
+  }
+
+  isLiked() {
+    return this._isLiked;
+  }
+
+  // setIsLiked(isLiked) {
```

Comment:
Needs correction: please uncomment this method and use it in index.js after updating likes in `then` block